README.md

```diff
@@ -1,2 +1,4 @@
 # growingGeometry
 This is a basic render created using Three JS in which a 3D object grows along all axis while rotating along the XYZ plane.
+
+How will your 2D screen show me 3D?
```

Comment:
We can color the geometrical figure and make light fall on it at an angle so as to get a perception of depth. As far as viewing something in 3D goes (for example in 3D movies), this render has its limitations for the human eye that can see things only in 2D.